eng/Versions.props

```diff
@@ -12,127 +12,77 @@
+    <SystemThreadingChannelsVersion>4.7.1</SystemThreadingChannelsVersion>
+
+    <!-- Other product dependencies -->
     <ApacheArrowVersion>2.0.0</ApacheArrowVersion>
-    <SystemTextEncodingVersion>4.3.0</SystemTextEncodingVersion>
-    <MicrosoftCSharpVersion>4.5.0</MicrosoftCSharpVersion>
-    <SystemCompositionVersion>1.2.0</SystemCompositionVersion>
-    <FluentAssertionVersion>5.10.2</FluentAssertionVersion>
-    <ApprovalTestVersion>5.4.7</ApprovalTestVersion>
+    <GoogleProtobufVersion>3.19.4</GoogleProtobufVersion>
+    <LightGBMVersion>2.3.1</LightGBMVersion>
+    <MicrosoftCodeAnalysisAnalyzersVersion>3.3.0</MicrosoftCodeAnalysisAnalyzersVersion>
+    <MicrosoftCodeAnalysisCSharpVersion>3.9.0</MicrosoftCodeAnalysisCSharpVersion>
+    <MicrosoftDotNetInteractiveFormattingVersion>1.0.0-beta.22103.1</MicrosoftDotNetInteractiveFormattingVersion>
+    <MicrosoftDotNetInteractiveVersion>1.0.0-beta.22103.1</MicrosoftDotNetInteractiveVersion>
+    <MicrosoftMLFeaturizersVersion>0.4.1</MicrosoftMLFeaturizersVersion>
+    <MicrosoftMLOnnxRuntimeVersion>1.10.0</MicrosoftMLOnnxRuntimeVersion>
+    <MlNetMklDepsVersion>0.0.0.12</MlNetMklDepsVersion>
+    <MSTestTestAdapterVersion>2.1.0</MSTestTestAdapterVersion>
+    <MSTestTestFrameworkVersion>2.1.0</MSTestTestFrameworkVersion>
+    <NewtonsoftJsonVersion>10.0.3</NewtonsoftJsonVersion>
+    <ParquetDotNetVersion>2.1.3</ParquetDotNetVersion>
+    <SharpZipLibVersion>1.3.3</SharpZipLibVersion>
+    <TensorflowDotNETVersion>0.20.1</TensorflowDotNETVersion>
+    <TensorFlowMajorVersion>2</TensorFlowMajorVersion>
+    <TensorFlowVersion>2.3.1</TensorFlowVersion>
+
     <!-- Build/infrastructure Dependencies -->
-    <PublishSymbolsPackageVersion>1.0.0-beta-62824-02</PublishSymbolsPackageVersion>
     <CodecovVersion>1.12.4</CodecovVersion>
     <CoverletCollectorVersion>1.2.1</CoverletCollectorVersion>
-    <ReportGeneratorVersion>4.3.6</ReportGeneratorVersion>
-    <MicrosoftDotNetApiCompatPackageVersion>1.0.0-beta.19225.5</MicrosoftDotNetApiCompatPackageVersion>
+    <CoverletMsbuildVersion>2.9.0</CoverletMsbuildVersion>
+    <MicrosoftCodeAnalysisCSharpInternalAnalyzerVersion>3.3.1</MicrosoftCodeAnalysisCSharpInternalAnalyzerVersion>
+    <MicrosoftCSharpVersion>4.5.0</MicrosoftCSharpVersion>
+    <MicrosoftDotNetApiCompatVersion>1.0.0-beta.19225.5</MicrosoftDotNetApiCompatVersion>
     <MicrosoftSourceLinkVersion>1.1.0-beta-20206-02</MicrosoftSourceLinkVersion>
-    <BuildToolsPackageVersion>3.0.0-preview4-04926-01</BuildToolsPackageVersion>
+    <MicrosoftSymbolUploaderBuildTaskVersion>1.1.225302</MicrosoftSymbolUploaderBuildTaskVersion>
+    <ReportGeneratorVersion>4.3.6</ReportGeneratorVersion>
+    <SystemCompositionVersion>1.2.0</SystemCompositionVersion>
+
     <!-- Test-only Dependencies -->
+    <ApprovalTestsVersion>5.4.7</ApprovalTestsVersion>
     <BenchmarkDotNetVersion>0.12.0</BenchmarkDotNetVersion>
+    <DotNetRuntime21Version>2.1.0</DotNetRuntime21Version>
```

Comment:
Do we even need to keep this one anymore?

Reply:
I can see about removing it in a follow up PR.  This PR is just organization.